log/logger.go

```diff
@@ -0,0 +1,22 @@
+package log
```

Comment:
Would it be better to call the package  `logger` to avoid naming from the standard log package?

Reply:
We can call it `logger` but this decision shouldn't be because the standard package is also called `log`. I called it `log` just to keep the name short. People will have to write it in every constructor so I thought its better if we do something like
```
struct {
l log.Logger
}
```
instead of 
```
struct {
l logger.Logger
}
```
What do you think?

Reply:
Ok cool. let's keep the same.